modules/apps/wiki/wiki-web/src/main/resources/META-INF/resources/wiki_admin/configuration_browse.jsp

```diff
@@ -0,0 +1,153 @@
+<%--
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+--%>
+
+<%@ include file="/wiki/init.jsp" %>
+
+<%
+MailTemplatesHelper mailTemplatesHelper = new MailTemplatesHelper(wikiRequestHelper);
+
+WikiConfigurationDisplayContext wikiConfigurationDisplayContext = new WikiConfigurationDisplayContext(request, renderRequest, renderResponse);
+
+portletDisplay.setShowBackIcon(true);
+portletDisplay.setURLBack(wikiConfigurationDisplayContext.getBackURL());
+portletDisplay.setURLBackTitle("wiki");
+%>
+
+<clay:container-fluid
+	cssClass="container-form-lg"
+>
+	<clay:row>
+		<clay:col
+			lg="3"
+		>
+			<c:if test="<%= PortalUtil.isRSSFeedsEnabled() %>">
+				<p class="c-mb-1 sheet-tertiary-title text-2 text-secondary">
+					<liferay-ui:message key="settings" />
+				</p>
+
+				<clay:vertical-nav
+					verticalNavItems="<%= wikiConfigurationDisplayContext.getSettingsVerticalNavItemList() %>"
+				/>
+			</c:if>
+
+			<p class="c-mb-1 sheet-tertiary-title text-2 text-secondary">
+				<liferay-ui:message key="notifications" />
+			</p>
+
+			<clay:vertical-nav
+				verticalNavItems="<%= wikiConfigurationDisplayContext.getNotificationsVerticalNavItemList() %>"
+			/>
+		</clay:col>
+
+		<clay:col
+			lg="9"
+		>
+			<h2>
+				<%= wikiConfigurationDisplayContext.getTitle() %>
+			</h2>
+
+			<liferay-portlet:actionURL portletConfiguration="<%= true %>" var="configurationActionURL">
+				<portlet:param name="navigation" value="<%= wikiConfigurationDisplayContext.getNavigation() %>" />
+				<portlet:param name="serviceName" value="<%= WikiConstants.SERVICE_NAME %>" />
+				<portlet:param name="settingsScope" value="group" />
+			</liferay-portlet:actionURL>
+
+			<aui:form action="<%= configurationActionURL %>" method="post" name="fm">
+				<clay:sheet
+					cssClass="c-mb-4 c-mt-4 c-p-0"
+					size="full"
+				>
+					<h3 class="c-pl-4 c-pr-4 c-pt-4 sheet-title">
+						<clay:content-row
+							verticalAlign="center"
+						>
+							<clay:content-col>
+								<%= wikiConfigurationDisplayContext.getSubtitle() %>
+							</clay:content-col>
+						</clay:content-row>
+					</h3>
+
+					<aui:input name="<%= Constants.CMD %>" type="hidden" value="<%= Constants.UPDATE %>" />
+					<aui:input name="redirect" type="hidden" value="<%= configurationActionURL %>" />
+
```

Comment:
Hi! I removed this because I did the validation in the otherwise option. Where I added the input I also added the validator for each one. Let me know your thoughts 😄